### PR TITLE
Focus 297

### PR DESCRIPTION
**Pull request by rbadamsjr**
Currently the Quantity and the data is not connected to a endpoint in  because one has not been made.FOCUS-309 was created to build out the backend in django

### CODE REVIEW

**Comment by rbadamsjr** [comment]
Final change has been pushed and i squashed my commits, if you do Arushi's first I can rebase mine because I think it will be a merge conflict on tha app/api/apiSlice.js files

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,5 @@ frontend/build
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+./dashboard.css*
```
Comment (jsnkle) [inline .gitignore:34]: this should be a partial so it doesn't compile into css and have to be ignored

File: frontend/package.json [hunk 2]
```diff
@@ -11,7 +11,9 @@
     "@popperjs/core": "^2.11.5",
     "@reduxjs/toolkit": "^1.8.3",
     "bootstrap": "^5.2.0",
+    "chart.js": "^3.9.1",
```
Comment (jsnkle) [inline frontend/package.json:14]: is this library needed in addition to react-chartjs-2?

File: frontend/src/app/api/apiSlice.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+const baseQuery = fetchBaseQuery({
+	baseURL: "https://focus-platform-ag.herokuapp.com",
+	credentials: "include",
```
Comment (jsnkle) [inline frontend/src/app/api/apiSlice.js:5]: Arushi did this a little differently than I did initially and functioning so I merged to master. Let's use hers for now although I think we have an opportunity in the future for some refactoring.

File: frontend/src/pages/Dashboard.js [hunk 4]
```diff
@@ -1,7 +1,52 @@
...
+		data: initiatives,
+		isSuccess: initiativesLoaded,
+		isError: isInitiativesError,
+		error: initiativesErrorMsg,
+	} = useGetOverallInitiativesQuery();
+
 	return (
 		<>
-			<h1 style={{ margin: "0" }}>Dashboard</h1>
+			{/*	<h1 style={{ margin: "0" }}>Dashboard</h1> */}
```
Comment (jsnkle) [inline frontend/src/pages/Dashboard.js:36]: You can go ahead and remove this altogether... was just a placeholder.

File: frontend/src/app/store.js [hunk 5]
```diff
@@ -1,8 +1,12 @@
 import { configureStore } from "@reduxjs/toolkit";
+import { apiSlice } from "./api/apiSlice";
 import drawerReducer from "features/ui/side-nav/drawerSlice";
 
 export const store = configureStore({
 	reducer: {
 		drawer: drawerReducer,
+		[apiSlice.reducerPath]: apiSlice.reducer
 	},
+	middleware: (getDefaultMiddleware) =>
```
Comment (jsnkle) [inline frontend/src/app/store.js:10]: good catch! we definitely need this.

**Review by jsnkle — CHANGES_REQUESTED** [review]
Let's simplify this naming a bit:
- feature = snapshot/Snapshot.js
- styling = _snapshot.scss
- slice = snaphotSlice.js
- const = snapshotApiSlice

**Comment by rbadamsjr** [comment]
Closed pr ... making a new one against develop

**Pull request closed without merging**